src/Windowing/Silk.NET.Windowing.Common/Interfaces/IWindow.cs

```diff
@@ -33,6 +33,17 @@ public interface IWindow : IWindowProperties, IWindowHost, IView
         /// </summary>
         new bool IsClosing { get; set; }
         
+        /// <summary>
+        /// Gets the distances in screen coordinates from the edges of the content area to the corresponding edges of
+        /// the full window.
+        /// </summary>
+        /// <remarks>
+        /// Because these are distances and not coordinates, they are always zero or positive.
+        /// </remarks>
+        /// <seealso cref="WindowExtensions.GetFullSize"/>
+        // This is in IWindow because it can't be configured.
+        Rectangle BorderSize { get; }
```

Comment:
This is, theoretically, a breaking change. Just noting, I think it's probably fine,

Reply:
What would it break?

Reply:
Theoretically `IWindow` is a public interface, and someone may have mocked/implemented it themselves (best case I can think of is for Unit testing) and this would break their code.